lib/irb/init.rb

```diff
@@ -235,6 +236,7 @@ def IRB.unset_measure_callback(type = nil)
       type_sym = type.upcase.to_sym
       IRB.conf[:MEASURE_CALLBACKS].reject!{ |t, | t == type_sym }
     end
+    IRB.conf[:MEASURE] = nil if IRB.conf[:MEASURE_CALLBACKS].empty?
```

Comment:
`measure :off, :measure_type` had a bug
```ruby
# lib/irb/cmd/measure.rb:20
when :off
  IRB.conf[:MEASURE] = nil
  IRB.unset_measure_callback(arg)
```
That disables all measure even if arg is specified.
```
irb> measure :foo
"FOO" is added
irb> measure :bar
"BAR" is added
irb> measure :off, :foo
# removes :foo and turns all measure off
irb> 42
# no measure
irb> measure
"TIME" added. (and BAR is re-enabled)
```